README.md

```diff
@@ -11,7 +15,7 @@
 The purpose of an inverted index is to allow user perform fast and full text search
 
 # **How can I get started with the project**
-* NA
+* You can view the application [here](https://indexinverted.herokuapp.com/)
 
 # **Technologies for the app**
```

Comment:
Make your READme more explanatory. You can provide background context and sample. Include more information like: how to use locally, run tests, contribute to the project and any other relevant information